### DISCUSSION TITLE
使用 iPhone 感觉不好的几点体验 — 码志

### DISCUSSION THREAD

**Discussion by mzlogin** [mzlogin/blog-comments#201] [Announcements]
https://mazhuang.org/2022/03/06/iphone-vs-xiaomi/ 

 上周有天晚上加完班下雨了，媳妇儿开车顺道带我回家，到家后过了一会儿她突然说，「对了，给你个东西」，然后从包里拿出一个黑盒子递给我，我正坐那刷沙雕视频，也没细看，见挺小一个盒子，随口问了一句「啥？充电宝啊？」，得到的是充满鄙夷的回复，「我们家需要那玩意儿吗，我买那干啥？」，接过来翻面一看，哦豁原来是 iPhone 13Pro。

**Comment by iwatchnetflix** [comment]
已经可以戴口罩解锁啦

**Comment by mzlogin** [comment]
> @iwatchnetflix
> 已经可以戴口罩解锁啦

确实～已升级，这个问题算解决啦～

**Comment by chen0331** [comment]
作者大大，文章下方的这个蓝框中的文档信息该怎么添加和修改啊，我是小白🤣

**Comment by mzlogin** [comment]
> @chen0331
> 作者大大，文章下方的这个蓝框中的文档信息该怎么添加和修改啊，我是小白🤣

我这个博客模板里，是修改 _includes/copyright.html 文件。

**Comment by gtming0508** [comment]
NFC交通卡 可以开通京津冀互联互通卡   全国大部分城市通用

**Comment by mzlogin** [comment]
> @gtming0508
> NFC交通卡 可以开通京津冀互联互通卡   全国大部分城市通用

嗯～但武汉这方面有点奇葩，这个卡，公交部分可用，地铁不可用……也不知道啥时候能整合下。
